src/Categories/Category/Monoidal/Instance/Rels.agda

```diff
@@ -0,0 +1,81 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Monoidal.Instance.Rels where
+
+open import Data.Empty.Polymorphic
+import Data.Product as ×
+open import Data.Sum
+open import Function
+open import Level
+open import Relation.Binary.PropositionalEquality
+
+open import Categories.Category.Cartesian
+open import Categories.Category.Cocartesian
+open import Categories.Category.Instance.Rels
+
+module _ {o ℓ} where
+
+  Rels-Cartesian : Cartesian (Rels o ℓ)
```

Comment:
Perhaps a comment that this is with respect to the disjoint union operation? I naively expected product, so was surprised.